packages/components/addon/components/hds/table/index.js

```diff
@@ -8,7 +8,7 @@ import { tracked } from '@glimmer/tracking';
 import { action } from '@ember/object';
 import { assert } from '@ember/debug';
 
-const DENSITIES = ['short', 'medium', 'tall'];
+export const DENSITIES = ['short', 'medium', 'tall'];
```

Comment:
Let me know if you don't like this and want me to revert it.

Reply:
No problem at all, it's a standard pattern in our repo.